### ISSUE TITLE
Top-level rems can't be deleted using the usual delete button

### ISSUE THREAD

**Issue by marcothesis** [remnoteio/remnote-issues#889]
**Tried to delete Top-level rem with delete button from menu as well as keyboard shortcut.**
---
![Screenshot 2021-02-24 at 5 07 22 PM](https://user-images.githubusercontent.com/10230857/108978389-7e537b80-76c4-11eb-977b-f74a8646d266.png)
---
**Top-level rem is still alive and well.**
---
![Screenshot 2021-02-24 at 5 07 58 PM](https://user-images.githubusercontent.com/10230857/108978640-c4a8da80-76c4-11eb-93bc-e3277f8c4681.png)
---
![Screenshot 2021-02-24 at 5 08 17 PM](https://user-images.githubusercontent.com/10230857/108978651-c8d4f800-76c4-11eb-91d5-ba38f2148096.png)
---
**At the moment, it seems only way is to track Top-level rem via document sidebar and delete it from there.**
---
![Screenshot 2021-02-24 at 5 10 18 PM](https://user-images.githubusercontent.com/10230857/108978887-0afe3980-76c5-11eb-8b94-7e08cfc6733c.png)

**Comment by mfranzs** [comment]
Thanks for reaching out. I'm having some trouble reproducing this. Does this happen for all top-level Rem?

**Comment by marcothesis** [comment]
For newly created top-level rems, I almost always fail to delete them the usual method for the first 1-2 tries. The reason I say 'almost' is because, when it finally works, I am able to delete them accordingly with 1 try for the other top-level rems.

For older top-level rems in my knowledge base, this does not work even after trying several times.

The only fool-proof way to delete them, as mentioned, is via the document sidebar.

**Comment by mfranzs** [comment]
Does the "Are you sure you want to delete this document?" popup appear even when the deletion fails?

**Comment by marcothesis** [comment]
Yes, "Are you sure you want to delete this document?" popup will still appear as per normal. But after clicking OK, I can still find the rem via document sidebar or CMD + P.

**Comment by drkvj** [comment]
I am not able to reproduce this as well

**Comment by marcothesis** [comment]
I just tried again and the issue still persists.

1 - create a new document via documents folder
2 - click documents folder and go back into created document
3 - click delete from top right corner menu
4 - click OK on popup confirmation to delete
5 - check inside the documents folder, it is still there
6 - 1st delete will fail but repeat step 2-4 again and with luck, it can be deleted 2nd, 3rd times etc....

As usual if my remnote freeze/crash because I tried to backspace delete a rem/document or I could not delete document via the top right corner menu, the only reliable way is to delete within documents folder itself.

**Comment by drkvj** [comment]
interesting, now I was able to reproduce it. 

I think the problem is only for documents that were created using the documents table. Thanks for helping us find this. 
CC: @mfranzs

**Comment by marcothesis** [comment]
I don't think this only applies to documents created via the documents table. I would never have created a document there if it wasn't for this test. I have encountered this bug in several instances but I don't think people often realize it exist if they don't check the documents table often.

I just created new references via cmd + p and then cmd + enter. Step 2 to 6 above still applies.

**Comment by drkvj** [comment]
Thanks for the added info Marcus, we will certainly take a look into this. I have been noticing this behaviour off lately as well.

**Comment by drkvj** [comment]
Another user has reported about the inability to delete documents with one try here : #1078
